src/components/Table/Table.stories.tsx

```diff
@@ -2,12 +2,21 @@ import { Table } from "./Table";
 
 const headers = [{ label: "Company" }, { label: "Contact" }, { label: "Country" }];
 const rows = [
-  [{ label: "Alfreds Futterkiste" }, { label: "Maria Anders" }, { label: "Germany" }],
-  [
-    { label: "Centro comercial Moctezuma" },
-    { label: "Francisco Chang" },
-    { label: "Mexico" },
-  ],
+  {
+    cells: [
```

Comment:
I find the name cells weird. Why not just `items`?

Reply:
I named it cells inorder to have the table cell concept be visible to the user using it
I can make it items @fneves Should I

Reply:
I would prefer items.